styles/style.css

```diff
@@ -8,22 +8,150 @@
   font-weight: 400;
 }
 
+.hovered:hover {
+  opacity: 0.7;
+}
+
 /*_____________________TOP-SECTION___________________________*/
 
 .top-section {
-  height: 200px;
-  background: #10367c;
+  background-image: url(../images/Top-section-castelmezzano.jpg);
```

Comment:
According to your hometask, the height of your top-side block must be at least like height of the screen. You can use property min-height: 100 vh. VH  - it is visual height.
 You can read more about  it here
 https://html5book.ru/edinicy-izmereniya-vh-vw-vmin-vmax/